docs2/site/docs/migrations/migration4.md

```diff
@@ -2,43 +2,87 @@
 
 ## New Features
 
-* Extension methods to configure authorization requirements for GraphQL elements: types, fields, schema.
-* New property `GraphQL.Introspection.ISchemaComparer ISchema.Comparer { get; set; }`
-* New property `IResolveFieldContext.ArrayPool`
+### Improved Performance
+
+GraphQL.NET 4.0 has been highly optimized, typically executing queries at least 50% faster while also providing a 65% memory reduction. Small queries have been measured to run as much as twice as fast as previously. A cached query executor is also provided, which can reduce execution time another 20% once the query has been parsed (disabled by default). Variable parsing is also improved to run about 50% faster, and schema build time is now about 20x faster than previously and requires 1/25th the amount of memory.
```

Comment:
> 1/25th the amount of memory

Where does this data come from?

Reply:
I ran tests just now.

Reply:
😮 😮 😮

Reply:
100,000 bytes vs 2.6mb

Reply:
Disabling xml comments was huge

Reply:
I’ll figure out the merge tool and post the results when I have time